pandas/tests/indexes/ranges/test_setops.py

```diff
@@ -446,8 +445,8 @@ def test_symmetric_difference(self):
         tm.assert_index_equal(result, expected)
 
         result = left.symmetric_difference(right[1:])
-        expected = Int64Index([1, 2, 3, 4, 5, 6, 7, 8, 9, 11, 12, 13, 14])
-        tm.assert_index_equal(result, expected)
+        expected = Index([1, 2, 3, 4, 5, 6, 7, 8, 9, 11, 12, 13, 14])
+        tm.assert_index_equal(result, expected, exact=True)
```

Comment:
Just confirming that `exact=True` still checks the exact Index class as well. (The docs still mention Int64Index but not sure if any changes were needed)

Reply:
`tm.assert_index_equal` works as expected, e.g.

```python
>>> import pandas as pd
>>> import pandas._testing as tm
>>> idx = pd.Index([*range(1, 15)])
>>> ri = pd.RangeIndex(1, 15)
>>> tm.assert_index_equal(idx, ri, exact=True)
AssertionError: Index are different

Index classes are different
[left]:  NumericIndex([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14], dtype='int64')
[right]: RangeIndex(start=1, stop=15, step=1)
```

The `_testing` module has to be cleaned up also, I'll do that soon.